Eos/Fuego/EOS.H

```diff
@@ -220,6 +220,14 @@ PYT2RE(
     E += Y[n] * ei[n];
 }
 
+AMREX_GPU_HOST_DEVICE
+AMREX_FORCE_INLINE
+void
+TYP2R(amrex::Real T, amrex::Real Y[], amrex::Real P, amrex::Real &R) 
+{
+  CKRHOY(&P, &T, Y, &R);
+}
+
```

Comment:
Don't you need to add this to the other EOS's?

Reply:
True, just created a similar function for GammaLaw and actually change the name of the function to be more consistent with the rest of EOS names.